packages/modal/src/resources/modal-util.ts

```diff
@@ -0,0 +1,28 @@
+import { UxModal } from '../ux-modal';
+
+export function findModal(item: HTMLElement | null): UxModal | null {
+  let element = item;
+  if (element === null) return null;
+  while (element.tagName !== 'BODY' && element.tagName !== 'UX-DRAWER') {
```

Comment:
Can we have some comments describing what would happen when we hit a `ux-drawer` here? So the reason why the check against `ux-drawer` can be easier to understand

Reply:
This is a mistake (left from a previous version of this code). Thanks for catching it, will simply remove.